server/middlewares/csp.js

```diff
@@ -14,7 +14,6 @@ export default function csp() {
   return helmet.contentSecurityPolicy({
     directives: {
       defaultSrc: trusted.concat([
-        '*.optimizely.com',
```

Comment:
According to https://help.optimizely.com/Troubleshoot_Problems/Update_your_site's_Content_Security_Policies_(CSP)

We need optimizely to be in the default. The only change is needed is to change this line to use https